modules/nw-creating-secured-routes.adoc

```diff
@@ -82,8 +80,8 @@ spec:
       -----END CERTIFICATE-----
 ----
 
-. Verify that the DNS entry for `www.example.test` points to the router
-instance(s) and that the route to your domain is available.
+. Verify that the DNS entry for `www.example.test` points to the ingress
+controller instance(s) and that the route to your domain is available.
```

Comment:
@openshift/sig-network-edge Instead of `ingress controller instance(s)`, should this be the ingress controller service ip?

Reply:
If this document applies generally irrespective of endpoint publishing strategy type, we cannot refer to the service.  On a cloud environment, it will be the `LoadBalancer`-type `Service`, but on UPI it will likely be an external load-balancer configured by the user.  How about "ingress controller endpoint"?

Reply:
Thanks! Updating